srcCpp03/PerftestTransport.cxx

```diff
@@ -886,4 +933,27 @@ bool PerftestTransport::parseTransportOptions(int argc, char *argv[])
     return true;
 }
 
+
+
+bool PerftestTransport::allowsMulticast()
+{
+    return (transportConfig.kind != TRANSPORT_TCPv4
+            && transportConfig.kind != TRANSPORT_TLSv4
+            && transportConfig.kind != TRANSPORT_WANv4
+            && transportConfig.kind != TRANSPORT_SHMEM
+            && useMulticast);
+}
+
+const std::string
```

Comment:
```
const std::string PerftestTransport::getMulticastAddr(
        const char *topicName)
{
```

Reply:
Is this so we use the same as in the old C++ implementation? @softwarejimenez

Reply:
Done.